test/e2e/e2e_helpers.py

```
@@ -0,0 +1,36 @@
+def ok(message: str) -> None:
+    print(f"[assert][pass] {message}")
+
+
+def check(condition: bool, success_message: str, failure_message: str) -> None:
+    if condition:
+        ok(success_message)
+        return
+    fail(failure_message)
+
+
+def make_initialize_payload(protocol: str, id: int = 1) -> dict:
```

Ruff (0.15.11)

[error] Function argument `id` is shadowing a Python builtin

(A002)